### PR TITLE
Adding intersect tests

### PR DESCRIPTION
**Pull request by galloj**
Solves #5

### CODE REVIEW

File: tests/intersect/2.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+U a b c d e
+S a b c
+S c d
+C intersect 1 2
```
Comment (JosefKuchar) [inline tests/intersect/2.txt:4]: Wrong line numbers
"Identifikátorem množiny je číslo řádku"
`C intersect 1 2` => `C intersect 2 3`

**Review by JosefKuchar — CHANGES_REQUESTED** [review]
Incorrect line number identifiers in multiple places

**Comment by galloj** [comment]
Fixed

**Comment by JosefKuchar** [comment]
Thanks!

**Pull request merged**